code/game/objects/items/weapons/switchtool.dm

```diff
@@ -61,9 +61,6 @@
 		choose_deploy(user)
 
 /obj/item/weapon/switchtool/attackby(var/obj/item/used_item, mob/user)
-	if(istype(used_item, removing_item) && /obj/item/weapon/switchtool/holo)
```

Comment:
I see what the error here was, but honestly I have a hard time blaming anyone except BYOND for this